pkg/pillar/cmd/agent.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"flag"
+	"github.com/lf-edge/eve/pkg/pillar/agentlog"
+	"github.com/lf-edge/eve/pkg/pillar/pidfile"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// AgentBase - Interface for all agents to use the AgentBase.
+// Each agent is expected to have a Context that implements
+// this interface.
+type AgentBaseIntf interface {
+	AgentBaseContext() *AgentBaseContext
+	AddAgentSpecificCommandLineFlags()
```

Comment:
Please pick one name and stick with it across the board. Seems like it is called "CLI" elsewhere so then don't call it CommandLine here.